odoo/addons/base/models/res_country.py

```diff
@@ -150,15 +150,53 @@ def _check_address_format(self):
                 except (ValueError, KeyError):
                     raise UserError(_('The layout contains an invalid format key'))
 
+    def _belongs_to_country_group(self, country_group):
+        """
+        This method determines whether the current country record is a member
+        of the specified country group with the code country_group_code.
+        """
+        if not self:
+            return False
+        self.ensure_one()
+        return country_group in self.country_group_ids.mapped('code')
+
+    def _get_country_group_codes(self):
+        """
+        Retrieve the codes of country groups associated with the current country.
+
+        This method returns a list of country group codes for the current country record,
+        but an empty list if there is no self.
+        """
+        if not self:
+            return ['']
+        self.ensure_one()
+        return self.country_group_ids.mapped('code') or ['']
```

Comment:
That's not an empty list.
```suggestion
            return []
        self.ensure_one()
        return self.country_group_ids.mapped('code') or []
```

If an empty *string* is required, please explain why in a comment.